### ISSUE TITLE
How can i run the program?

### ISSUE THREAD

**Issue by GottaSlay** [BlueGoliath/Goliath-Overclocking-Utility-FX#1]
I keep double-clicking the .jar but it wont open? I tried Java 8 runtime and Java 11 runtime.

**Comment by BlueGoliath** [comment]
Hi GottaSlay,

If you are using a Pascal(10 series) GPU then that is probably why. You can get an updated version(not built from git source) that will work for Pascal and (hopefully) late [here.](https://drive.google.com/open?id=1gh51QmogLTJBzBw7acdYrFBhEiWM8NZh)

Otherwise please make sure coolbits is set to 31.

This version includes its own Java 11 version built right in. To launch, go to the bin folder and run the "GoliathOUFX" file by doing:

./GoliathOUFX

from the command line.

Zipped and uploaded from Windows 10 so it may or may not work. If not Ill boot into Linux and rezip it.

(P.S. You need to run with sudo for changing power limit and voltage is not supported on Pascal(10 series).

**Comment by GottaSlay** [comment]
I do in fact have a Pascal card. So i tried to launch it by doing `./GoliathOUFX` which is when I received `./GoliathOUFX: 4: ./GoliathOUFX: ./java: Permission denied`

**Comment by BlueGoliath** [comment]
Yeah Windows messed it up somehow. Try [this](https://drive.google.com/open?id=1xudbenDDD2kZvshlhnBrMtrlQEqMey88) one.

Though to be honest, there isn't much of a point if you are running Pascal - once coolbits is set you can do basic core and memory clock via nvidia-settings and increasing powerlimit doesn't do much in most cases.

**Comment by GottaSlay** [comment]
Thanks! I think you should update the downloader to the new one you sent me. A linuxnewbie like me definitely prefers to use an overclocking-dedicated app with an easy-to-use GUI over messing with coolbits and terminal. Especially now when there are no apps supporting Nvidia overclocking and AMD does have an app. Plus there are a lot of "gamers" shifting over to Linux (including me). Who will be looking for an app like this. This is perfect and makes everything less of a hassle.

**Comment by BlueGoliath** [comment]
Coolbits is what enables overclocking, fan control, etc in Linux. It is not enabled by default and sadly must be enabled via the terminal. 

That said, once coolbits is set, you do have basic overclocking support via nvidia-settings which is installed by default along with the Nvidia graphics drivers. My OC utility is really only any better for cards that support overvolting or if you want to change the power limit. In fact, the utility utilizes the command line interface which provides extra functionality like overvolting on cards that support it. The CPU usage from constantly getting information from the command line will hurt game performance if you have the application open.
